lib/modules/webserver/index.js

```diff
@@ -64,6 +65,11 @@ class WebServer {
   }
 
   testPort(done) {
+    if (this.port === 0) {
```

Comment:
just curious here, can you do port 0? I am not able to assign to port 0.

Reply:
A port value of `0` is a signal to let the OS pick an available port.